admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -45,14 +58,24 @@ export const ApplicationPage: React.FunctionComponent = () => {
             name: name ?? '',
             description: description ?? ''
         });
```

Comment:
You could call the query here if you want to reload on create/update:
`mutate(...).then(eventTypesQuery.query);`

And probably you would have to reset the id/displayName/etc as well